lib/config/options/index.ts

```diff
@@ -1499,6 +1499,38 @@ const options: RenovateOptions[] = [
     env: false,
     supportedPlatforms: ['github'],
   },
+  {
+    name: 'osvVulnerabilityAlerts',
+    description: 'Use vulnerability alerts from osv.dev',
+    type: 'boolean',
+    default: false,
+    releaseStatus: 'unpublished',
+    // GitHub is not supported as it already provides vulnerability alerts
```

Comment:
I think it might make sense to support GitHub as well. I can imagine a few scenarios there this might be desired: 
- Users running GitHub Enterprise Server without dependency alerts enabled
- OSV contains additional vulnerabilities to GitHub alerts (not sure if that is the case?)
- In future maybe using a custom OSV database which might contain vulnerabilities for internal/custom packages

Reply:
I've decided to omit this for now, and just gate the feature behind one configuration option.